sdk/synapse/Microsoft.Azure.Synapse/Directory.Build.Data.props

```diff
@@ -0,0 +1,133 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why do you need this file?

Reply:
> Why do you need this file?

Please refer the first part for client libraries in the guide https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#project-structure

For short Client library projects needs to use the $(RequiredTargetFrameworks) and refer the client library eventhub I added this file

Reply:
> Why do you need this file?

Removed